src/components/ChannelCard.jsx

```diff
@@ -0,0 +1,85 @@
+import Image from 'next/image';
+import { Fragment } from 'react';
+
+import useStore from '../hooks/useStore';
+
+import StyledContainer from './StyledContainer';
+
+export default function ChannelCard({ channelItem }) {
+	const setCurrentItem = useStore(state => state.setCurrentItem);
+	const fetchChannelData = useStore(state => state.fetchChannelData);
+	const channelPlaylists = useStore(state => state.channelPlaylists);
+	const playlistVideos = useStore(state => state.playlistVideos);
+
+	return (
+		<StyledContainer key={channelItem.channelId} variant="channelCard">
+			<Image
+				src={channelItem.thumbnails?.high?.url}
+				alt={channelItem.channelTitle}
+				layout="fixed"
+				width={70}
+				height={70}
+			/>
```

Comment:
React's key prop is something you only need to put on elements when you map over an array 🔑

```suggestion
		<StyledContainer variant="channelCard">
			<Image
				src={channelItem.thumbnails?.high?.url}
				alt={channelItem.channelTitle}
				layout="fixed"
				width={70}
				height={70}
			/>
```